maro/cli/inspector/env_data_process.py

```diff
@@ -0,0 +1,201 @@
+    """Generate summary info of current scenario.
+    Different scenario has different data features.
+    e.g. cim has multiple epochs while citi_bike only has one.
+    Each scenario should be treated respectively.
+
+    Args:
+        scenario (enumerate): Current scenario.
+        source_path (str): Data folder path.
+        prefix (str): Prefix of data folders.
+
+    """
+    ports_header = ["capacity", "empty", "full", "on_shipper", "on_consignee", "shortage", "booking", "fulfillment"]
+    # vessels_header = ["capacity", "empty", "full", "remaining_space", "early_discharge"]
+    stations_header = ["bikes", "shortage", "trip_requirement", "fulfillment", "capacity"]
+    dbtype_list_all = os.listdir(source_path)
+    temp_len = len(dbtype_list_all)
+    dbtype_list = []
+    for index in range(0, temp_len):
+        if os.path.exists(os.path.join(source_path, f"{prefix}{index}")):
+            dbtype_list.append(os.path.join(source_path, f"{prefix}{index}"))
+
+    if scenario == GlobalScenarios.CIM:
+        _init_csv(os.path.join(source_path, Gfiles.ports_sum), ports_header)
+        # _init_csv(vessels_file_path, vessels_header)
+        ports_sum_dataframe =\
+            pd.read_csv(os.path.join(source_path, Gfiles.ports_sum), names=ports_header)
+        # vessels_sum_dataframe = pd.read_csv(vessels_file_path, names=vessels_header)
+    else:
+        _init_csv(os.path.join(source_path, Gfiles.stations_sum), stations_header)
+    if scenario == GlobalScenarios.CIM:
+        i = 1
+        for i in tqdm.tqdm(range(len(dbtype_list))):
+            dbtype = dbtype_list[i]
+            dir_epoch = os.path.join(source_path, dbtype)
+            if not os.path.isdir(dir_epoch):
+                continue
+            _summary_append(
+                scenario, dir_epoch, "ports.csv", ports_header,
+                ports_sum_dataframe, i, os.path.join(source_path, Gfiles.ports_sum))
+            # _summary_append(dir_epoch, "vessels.csv", vessels_header, vessels_sum_dataframe, i,vessels_file_path)
+            i = i + 1
+    elif scenario == GlobalScenarios.CITI_BIKE:
+        data = pd.read_csv(os.path.join(source_path, f"{prefix}0", "stations.csv"))
+        data = data[["bikes", "trip_requirement", "fulfillment", "capacity"]].groupby(data["name"]).sum()
+        data["fulfillment_ratio"] = list(
+            map(lambda x, y: float("{:.4f}".format(x / (y + 1 / 1000))), data["fulfillment"],
+                data["trip_requirement"]))
+        data.to_csv(os.path.join(source_path, Gfiles.stations_sum))
+
+
+def _get_holder_name_conversion(scenario: enumerate, source_path: str, conversion_path: str):
+    """ Generate a CSV File which indicates the relationship between index and holder"s name.
+
+    Args:
+        scenario (enumerate): Current scenario. Different scenario has different type of mapping file.
+        source_path (str): Data folder path.
+        conversion_path (str): Path of origin mapping file.
```

Comment:
According to ``conversion_path = os.path.join(source_path, conversion_path)``, why not ``conversion name``?